### PR TITLE
Upgrade gems

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

**Comment by pftg** [comment]
Deployed on https://jt-pr296-pftg.surge.sh

**Comment by pftg** [comment]
@andriyParashchuk @obliviusm @sweatC please review on http://jt-pr296-pftg.surge.sh/

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "gulp": "~3.9.1",
     "gulp-imagemin": "~4",
-    "imagemin-pngcrush": "~5"
+    "imagemin-pngcrush": "~5",
+    "surge": "^0.20.1"
```
Comment (obliviusm) [inline package.json:23]: 👍

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    }
+};
```
Comment (obliviusm) [inline .eslintrc.js]: no newline

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            4
```
Comment (obliviusm) [inline .eslintrc.js]: we use indentation 2

**Review by obliviusm — CHANGES_REQUESTED** [review]
@pftg do we need eslint? 
anyway your rules give too much errors in `application.js`

maybe separate pr for linting with lint command and fixed errors?

**Comment by pftg** [comment]
@obliviusm please re-review

**Comment by obliviusm** [comment]
@pftg 
```
eslint .

/jetthoughts.github.io/_assets/javascripts/application.js
  213:3  error  Parsing error: The keyword 'const' is reserved

/jetthoughts.github.io/_assets/javascripts/form.js
   1:1   error  '$' is not defined                no-undef
   2:21  error  '$' is not defined                no-undef
   3:17  error  '$' is not defined                no-undef
   4:22  error  '$' is not defined                no-undef
  22:23  error  'jsn' is defined but never used   no-unused-vars
  28:24  error  'form' is defined but never used  no-unused-vars
  29:5   error  '$' is not defined                no-undef
  44:16  error  '$' is not defined                no-undef
  51:16  error  '$' is not defined                no-undef
  64:16  error  '$' is not defined                no-undef
  69:5   error  '$el' is not defined              no-undef
  69:41  error  '$' is not defined                no-undef
  69:49  error  '$' is not defined                no-undef
  70:5   error  '$el' is not defined              no-undef
  71:18  error  '$' is not defined                no-undef
  84:16  error  '$' is not defined                no-undef

/jetthoughts.github.io/gulpfile.js
  1:1  error  Parsing error: The keyword 'const' is reserved

✖ 18 problems (18 errors, 0 warnings)

```

**Comment by pftg** [comment]
let's fix this later

**Comment by obliviusm** [comment]
@pftg i added few ignores to make linter pass. is it ok?

**Pull request merged**